database/buildSchema/26A_review_assignment.sql

```diff
@@ -15,6 +15,7 @@ CREATE TABLE public.review_assignment (
 	trigger public.trigger,
 	time_created timestamptz default current_timestamp,
 	level integer,
+-- 	level_id integer references public.template_stage_review_level(id),
```

Comment:
This is very bizarre. I've left it commented out for now, but if I try and make this foreign key reference, it creates a GraphQL schema problem, and the `templateActionsUsingId` and `templatePermissionsUsingId` create nodes are no longer available and the create template mutations fail. No idea why, since this shouldn't affect those nodes at all. Maybe it creates a circular reference somewhere?

Anyway, I don't think we really need it here. We only need to know the level number most of the time, a reference to the original record is not really necessary.

Reply:
I think it might be conflicting with the field above `level` because of the types which get generated by GraphQL. Can you try renaming the one above to `level_number` or just removing it?

Reply:
Okay, I've renamed "level" to "level_number" and the level_id now seems to work fine. Thank.

However, changing that field name required quite a bit of updating to other tables/data/actions, so the number of files changed here is now quite a bit more. I think I got them all. I'm planning to thoroughly check all the Actions are working properly as part of the "create company/logo" PR that I've almost finished, so I'll pull these changes into there before publishing once this is approved.

b4f0da5a